README.md

```diff
@@ -103,11 +103,11 @@ If it has changed, you can rebuild the glowfic image with:
 
 Now, update the gems used for the project:
 
-*   `docker-bin/bundler`
+*   `bin-docker/bundle install --gemfile /code/Gemfile`
```

Comment:
Hm this has worked for me in the past without the gemfile argument, does yours error without it?

Reply:
Looks like no!